.github/workflows/tests.yml

```
@@ -29,16 +29,16 @@ jobs:
       RAILS_ENV: test
       PGUSER: discourse
       PGPASSWORD: discourse
-      USES_PARALLEL_DATABASES: ${{ matrix.build_type == 'backend' || matrix.build_type == 'system' }}
+      USES_PARALLEL_DATABASES: ${{ matrix.build_type == 'backend' || matrix.build_type == 'system' || matrix.build_type == 'system.chat' }}
       CAPBYARA_DEFAULT_MAX_WAIT_TIME: 4
 
     strategy:
       fail-fast: false
 
       matrix:
-        build_type: [backend, frontend, system, annotations]
+        build_type: [backend, frontend, system, annotations, system.chat]
         target: [core, plugins]
-        ruby: ['3.2']
+        ruby: ["3.2"]
```

imo I'd be better to add a single new `target` for the `system` build_type:

```suggestion
        ruby: ["3.2"]
        include:
          - build_type: system
          - target: chat
```

Then you'd have to modify only the "Plugin System Tests" step (and

---

I was just trying something, sorry should have been draft